int16 gui_cmd_0
int16 gui_cmd_1
int16 gui_cmd_2
int16 gui_cmd_3
int16 gui_cmd_4
int16 gui_cmd_5
int16 gui_cmd_6
int16 gui_cmd_7